app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/reimbursement/CheckReimbursementClaimController.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.reimbursement
+
+import com.google.inject.Inject
+import play.api.Configuration
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.cache.SessionCache
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.actions.{AuthenticatedAction, SessionDataAction, WithAuthAndSessionDataAction}
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.{SessionDataExtractor, SessionUpdates}
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.utils.Logging
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+
+class CheckReimbursementClaimController @Inject() (
+  val authenticatedAction: AuthenticatedAction,
+  val sessionDataAction: SessionDataAction,
+  val sessionCache: SessionCache,
+  cc: MessagesControllerComponents,
+  val config: Configuration
+) extends FrontendController(cc)
+    with WithAuthAndSessionDataAction
+    with Logging
+    with SessionDataExtractor
+    with SessionUpdates {
+
+  def showReimbursementClaim(): Action[AnyContent] = Action {
+    Ok("implementation todo ....")
+  }
+
```

Comment:
Thinking maybe we can have temp continue button to go to single journey duties. I think @NarsiAgaram would be extremely happy.

Reply:
Yeah, I will add these in the next PR now. Only one more form to do and then we can complete the mini-flow.